Extension/src/LanguageServer/client.ts

```diff
@@ -2756,7 +2763,7 @@ export class DefaultClient implements Client {
                     }
                 }
                 sanitized.push({
-                    uri: item.uri.toString(),
+                    uri: uri,
```

Comment:
If the variable has the same name as the field, you can just put `uri` here instead of `uri: uri`